starter-code/css/main.css

```diff
@@ -0,0 +1,197 @@
+.navbar {
```

Comment:
Prefer to add your own classes or id's to manipulate, instead of changing the bootstrap ones directly.